docs/source/pre_application/digital_ocean_tutorial.rst

```diff
@@ -0,0 +1,134 @@
+.. _digital-ocean-tutorial:
+
+======================================================================
+Tutorial for Creating a Threshold PRE Testnet Node using Digital Ocean
+======================================================================
+
+In this tutorial we're going to setup a Threshold PRE Node on the rinkeby testnet.
+There are a few pre-requisites before we can get started.
+First, we need to create accounts at `Digital Ocean <https://cloud.digitalocean.com/>`_ and `Infura <https://infura.io>`_.
+We also need at least 40,000T in our wallet.
+There are currently no Threshold faucets but if you ask in the `Discord <https://discord.gg/Threshold>`_ channel, someone will be happy to help you out.
+
+   .. note::
+
+      Ensure that you are using a wallet on the rinkeby testnet, don't use a mainnet address.
+
+
+
+Digital Ocean
+-------------
+All of our Digital Ocean configuration will be done automatically, but there are two local environment variables we need to set in order to make this work:
+- `DIGITALOCEAN_ACCESS_TOKEN` - Your Digital Ocean `access token <https://docs.digitalocean.com/reference/api/create-personal-access-token/>`_`
+- `DIGITAL_OCEAN_KEY_FINGERPRINT` - Your Digital Ocean `key fingerprint <https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-account/>`_.
+
+Follow those two blog posts and either `export` the variables or add them to your `~/.bashrc` file.
```

Comment:
I think some of the terms with simple quotes in this file, actually need double-quotes in RST format. E.g.:
```
  ``~/.bashrc``
```